src/components/button-toggle-group/button-toggle-group.stories.mdx

```diff
@@ -109,6 +109,36 @@ import ButtonToggleGroup from "carbon-react/lib/components/button-toggle-group";
   </Story>
 </Canvas>
 
+### With fullWidth
+
+When the `fullWidth` prop is `true`, the css property flex: auto is applied to each `Button`s 
+parent container, and each `Button`s label width is set to 100%. 
```

Comment:
This is really picky but I think this goes too much into implementation details. From the consumer's point of view the important thing is that the buttons expand to fill the full width of the container. So I'd say pretty much literally that, eg:

```
When the `fullWidth` prop is `true`, the buttons will expand in size to take up the full width of the container.
```